bundles/org.openhab.binding.growatt/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -439,6 +439,47 @@
 				<description>Total reactive energy supplied.</description>
 			</channel>
 
+			<!-- ============== CHANNELS ADDED IN NOVEMBER 2024 ============== -->
+
+			<!-- battery instantaneous measurements -->
+			<channel id="battery-voltage2" typeId="advanced-electric-voltage">
+				<label>Battery Voltage</label>
```

Comment:
Probably this should have a different label to distinguish it from channel `battery-voltage`?